packages/webapp/src/stories/Card/styles.module.scss

```diff
@@ -12,28 +12,11 @@
  *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
  */
```

Comment:
How did the GitHub diff algorithm come to the conclusion that this was a rename? 😂